subsys/usb/usb_c/usbc_pe_snk_states.c

```diff
@@ -1583,7 +882,7 @@ static void pe_drs_evaluate_swap_run(void *obj)
 /**
  * @brief PE_DRS_Send_Swap Entry state
  */
-static void pe_drs_send_swap_entry(void *obj)
+void pe_drs_send_swap_entry(void *obj)
```

Comment:
A lot of these states, for instance this one, are not sink-specific. Should they be in the common file?

Reply:
This was done in a later PR but should be done here. I've moved it to common

Reply:
I've removed the functions